src/snowcli/cli/warehouse/commands.py

```diff
@@ -15,6 +15,12 @@
 )
 
 
+# Empty callback is required to create command group with only one command
```

Comment:
I don't have a better idea at the moment. In the future we can think about an util function which will be responsible for creating a typer of specific type (for example: "typer for command group with only one command").

Reply:
More comments about it: https://github.com/Snowflake-Labs/snowcli/pull/385#discussion_r1328888405